tket/tests/TokenSwapping/Data/FixedCompleteSolutions.cpp

```diff
@@ -0,0 +1,2644 @@
+// Copyright 2019-2021 Cambridge Quantum Computing
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "FixedCompleteSolutions.hpp"
+
+#include <catch2/catch.hpp>
+
+namespace tket {
+namespace tsa_internal {
+namespace tests {
+
+FixedCompleteSolutions::FixedCompleteSolutions() {
```

Comment:
"I don't think random problem generation is particularly hard or interesting in this case ...." -- I agree, but it _does_ matter that the correctness of the table here should be verifiable by a third party. One day we may do some refactoring and get a test failure, and we need to know if the bug is in the code or in the tests. At the moment it would be difficult to tell.